### PR TITLE
Remove zipkin scribe receiver and run go mod tidy everywhere

### LINKED ISSUE #409: Update to Thrift v0.13.0 (fix scribe-go)
**Is your feature request related to a problem? Please describe.**
Updating to the latest core fails due to a breaking change in Thrift v0.13.0:
```
make otelcontribcol
GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_linux_amd64 -ldflags "-X github.com/open-telemetry/opentelemetry-collector-contrib/internal/version.GitHash=a8ed94b  -X go.opentelemetry.io/collector/internal/version.BuildType=release" ./cmd/otelcontribcol
# github.com/omnition/scribe-go/if/scribe/gen-go/scribe
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:78:20: not enough arguments in call to oprot.Flush
	have ()
	want (context.Context)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:102:15: assignment mismatch: 2 variables but error0.Read returns 1 values
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:148:28: cannot use &scribeProcessorLog literal (type *scribeProcessorLog) as type thrift.TProcessorFunction in assignment:
	*scribeProcessorLog does not implement thrift.TProcessorFunction (wrong type for Process method)
		have Process(int32, thrift.TProtocol, thrift.TProtocol) (bool, thrift.TException)
		want Process(context.Context, int32, thrift.TProtocol, thrift.TProtocol) (bool, thrift.TException)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:158:27: not enough arguments in call to processor.Process
	have (int32, thrift.TProtocol, thrift.TProtocol)
	want (context.Context, int32, thrift.TProtocol, thrift.TProtocol)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:166:13: not enough arguments in call to oprot.Flush
	have ()
	want (context.Context)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:183:14: not enough arguments in call to oprot.Flush
	have ()
	want (context.Context)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:196:14: not enough arguments in call to oprot.Flush
	have ()
	want (context.Context)
../../go/pkg/mod/github.com/omnition/scribe-go@v1.0.0/if/scribe/gen-go/scribe/scribe.go:210:23: not enough arguments in call to oprot.Flush
	have ()
	want (context.Context)
make: *** [Makefile:126: otelcontribcol] Error 2
```

Go types in https://github.com/Omnition/scribe-go were generated using old Thrift binaries (v0.9.3).

**Describe the solution you'd like**
Update `scribe-go` to Thrift v0.13.0 to unblock the update, and update its dependency in this repo.

**Describe alternatives you've considered**
`scribe-go` is being used by Zipkin Scribe receiver. We may remove it from the contrib repo to fix the issue.

**Additional context**
-

/cc @bogdandrutu @pjanotti

### PR DESCRIPTION
**Pull request by bogdandrutu**
Zipkin Scribe support was deprecated more than few years ago, and hard to find any documentation, found some PRs related in go-kit https://github.com/go-kit/kit/commit/48b40287851bb1630eb6d3007db16c82a34844b0#diff-bec2cc44db2f273eb2db7579a445378e

Edit: Scribe is not officially deprecated but not used that much, see https://groups.google.com/forum/#!topic/zipkin-user/0XN7G2gArUc

Fixes: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/409

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -27,6 +27,7 @@ contrib.go.opencensus.io/exporter/stackdriver v0.13.1/go.mod h1:z2tyTZtPmQ2HvWH4
 contrib.go.opencensus.io/resource v0.1.2 h1:b4WFJV8u7/NzPWHeTqj3Ec2AW8OGhtJxC/hbphIOvbU=
 contrib.go.opencensus.io/resource v0.1.2/go.mod h1:F361eGI91LCmW1I/Saf+rX0+OFcigGlFvXwEGEnkRLA=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
+git.apache.org/thrift.git v0.12.0/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg=
```
Comment (nilebox) [inline go.sum:30]: Is this some fork of Thrift? Looks like it appeared after removing
```
replace git.apache.org/thrift.git v0.12.0 => github.com/apache/thrift v0.12.0
```
from go.mod above.

`go mod graph` shows that OpenCensus depends on it:
```
go.opencensus.io@v0.19.1 git.apache.org/thrift.git@v0.12.0
```
which is coming from this exporter:
```
github.com/signalfx/opencensus-go-exporter-kinesis@v0.4.2 go.opencensus.io@v0.19.1
```

**Review by james-bebbington — APPROVED** [review]

**Review by nilebox — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (bogdandrutu) [reply go.sum:30]: It is moved from line 96. Don't ask me why `go mod tidy` decided on this change.

Comment (nilebox) [reply go.sum:30]: The latest Kinesis exporter release https://github.com/signalfx/opencensus-go-exporter-kinesis/releases/tag/v0.6.3 has a newer dependency on [OpenCensus v0.22.2](https://github.com/signalfx/opencensus-go-exporter-kinesis/blob/873a7ae3fb6d37cc78b8b692e4f1a8ceb89218bb/go.mod#L12) so updating it should eliminate this dependency.

Comment (bogdandrutu) [reply go.sum:30]: Sorry, that replace was given me an error after I deleted the scribe. That's why I removed it.

Comment (bogdandrutu) [reply go.sum:30]: @nilebox please send a PR after this.

**Pull request merged**